sktime/split/expandinggreedy.py

```diff
@@ -16,30 +17,37 @@
 class ExpandingGreedySplitter(BaseSplitter):
     """Splitter that uses all available data.
 
-    Takes an integer `test_size` that defines the number of steps included in the
+    Takes an integer ``test_size`` that defines the number of steps included in the
     test set of each fold. The train set of each fold will contain all data before
-    the test set. If the data contains multiple instances, `test_size` is
+    the test set. If the data contains multiple instances, ``test_size`` is
     _per instance_.
 
-    If no `step_length` is defined, the test sets (one for each fold) will be
-    adjacent, taken from the end of the dataset.
+    If no ``step_length`` is defined, the test sets (one for each fold) will be
+    adjacent and disjoint, taken from the end of the dataset.
 
-    For example, with `test_size=7` and `folds=5`, the test sets in total will cover
+    For example, with ``test_size=7`` and ``folds=5``, the test sets in total will cover
     the last 35 steps of the data with no overlap.
 
     Parameters
     ----------
-    test_size : int
-        The number of steps included in the test set of each fold.
+    test_size : int or float
+        If int: the number of steps included in the test set of each fold.
+            Formally, steps are consecutive ``iloc`` indices.
+        If float: the proportion of steps included in the test set of each fold,
+            as a proportion of the total number of index values.
+            Cave: not the ``loc`` proportion between start and end.
     folds : int, default = 5
         The number of folds.
     step_length : int, optional
         The number of steps advanced for each fold. Defaults to `test_size`.
+    reverse : bool, default = False
```

Comment:
just wondering, when would this reverse argument be useful? 

Isn't forecasting is about forecasting the future? why would test set be at the start of the data?

Reply:
it's for repurposing the splitter to slice from the start of the time series, it would notb e useful by itself.

Reply:
I removed this again, I think you are right, this was strange.